pkg/webhook/admission/pod/agent_injector.go

```diff
@@ -277,6 +288,30 @@ func (ag *AgentInjector) InjectAgent(pod *v1.Pod) error {
 		},
 	}
 
+	// If the Logger TLS bundle ConfigMap is specified, mount it
+	if injectLogger && ag.loggerConfig.TlsBundle != "" {
+		// Optional. If the ConfigMap is not found, this will not make the Pod fail
+		configMapVolume := v1.VolumeSource{
+			ConfigMap: &v1.ConfigMapVolumeSource{
+				LocalObjectReference: v1.LocalObjectReference{
+					Name: ag.loggerConfig.TlsBundle,
```

Comment:
Is the CA bundle configmap available in each namespace?

Reply:
Hi @yuzisun, addressing this question and the one [above](https://github.com/kserve/kserve/pull/3863#discussion_r1720545006).

The CA bundle's name is specified in the global ConfigMap.
When the agent is created, a ConfigMap from the same namespace as the InferenceService and with that matching name is mounted as a volume and used to configure the transport.
Since the previous behaviour (before this PR) was that `https` could be specified as a logger endpoint, although with no TLS configuration, the following is expected in order keep this backwards compatible with the previous behaviour:
- If the ConfigMap is _not_ present in the InferenceService's namespace and the logger endpoint is `https`, continue using that endpoint but with no TLS configured, i.e. the ConfigMap is optional and it won't cause the agent to fail if missing